.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Run Steampipe awstf compliance
+on: [pull_request,push]
```

Comment:
Can you please update the trigger to only trigger on the `main` branch (we won't have releases/* branches), e.g., https://github.com/turbot/steampipe-action-setup/blob/48a6f92adcd012d08af42b319d246005f158d051/.github/workflows/test.yml#L4-L6